xml/System/String.xml

```diff
@@ -14941,8 +14941,8 @@ The `TrimStart(System.Char[])` method removes from the current string all leadin
 
 The following example demonstrates the basic functionality of the <xref:System.String.TrimStart%2A> method:
 
-[!code-csharp[System.String.TrimStart#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.String.TrimStart/cs/sample.cs#1)]
-[!code-vb[System.String.TrimStart#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.String.TrimStart/vb/sample.vb#1)]
+:::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.String.TrimStart/cs/sample.cs" interactive="try-dotnet-method" id="Snippet1":::
```

Comment:
This code was supposed to work but we actually found a bug with the .NET interactive feature

```suggestion
:::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.String.TrimStart/cs/sample.cs" id="Snippet1":::
```